swap_meet/vendor.py

```diff
@@ -1,2 +1,57 @@
+from swap_meet.item import Item 
+
 class Vendor:
-    pass
+    def __init__(self, inventory = []):
+        self.inventory = inventory 
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item 
+
+    def remove(self, item):
+        if item in self.inventory: 
+            self.inventory.remove(item)
+            return item
+        else:
+            return False 
+
+    def get_by_category(self, category):
+        inventory_list = []
+        for item in self.inventory: 
+            if item.category == category:
+                inventory_list.append(item)
+        return inventory_list 
+
+    def swap_items(self, friend, my_item, their_item):
+        if their_item in friend.inventory and my_item in self.inventory:
+            self.add(their_item)
+            friend.add(my_item)
+            self.remove(my_item)
+            friend.remove(their_item)
```

Comment:
great job reusing methods you've already made!we can make this even shorter, though:

```suggestion
            self.add(friend.remove(their_item))
            friend.add(self.remove(my_item))
```
if you recall, the `remove` method returns the item that is removed, so we can call the `remove` method right inside the `add`